cmd/harmony/main.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/harmony-one/harmony/internal/genesis"
 	hmykey "github.com/harmony-one/harmony/internal/keystore"
 	"github.com/harmony-one/harmony/internal/memprofiling"
+	"github.com/harmony-one/harmony/internal/multibls"
```

Comment:
no, let's not put this into internal

Reply:
what would be the appropriate location?

Reply:
will move it under `github.com/harmony-one/harmony/multibls`